docs/tutorials/stateful-application/zookeeper.md

```diff
@@ -985,25 +985,20 @@ kubernetes-minion-group-a5aq
 kubernetes-minion-group-2g2d
 ```
 
-This is because the Pods in the `zk` StatefulSet contain a 
-[PodAntiAffinity](/docs/user-guide/node-selection/) annotation.
+This is because the Pods in the `zk` StatefulSet have a
+[PodAntiAffinity](/docs/user-guide/node-selection/) specified.
```

Comment:
link to /docs/user-guide/node-selection/#inter-pod-affinity-and-anti-affinity-beta-feature